vars/buildPlugin.groovy

```diff
@@ -9,6 +9,7 @@ def call(Map addonParams = [:])
 	def VERSIONS_VALID = [
 		'Leia': 'leia',
 		'Matrix': 'matrix',
+		'Nexus': 'nexus'
```

Comment:
Why no comma?

Reply:
The last in list, like on value `def PLATFORMS_VALID = [` below, thought to leave is also not 100% correct (maybe there json works with).

Reply:
This is not JSON, it's groovy.

Reply:
I change it, know thats done by groovy, but looks a bit in json format style to me.

Is updated.